app/views/patients/_form.html.slim

```diff
@@ -39,7 +39,7 @@
       h4 VA Status and Travel Eligibility
       .row
         .col-md-6
-          = f.input :assigned_vamc, :label => "Assigned VAMC", collection: Patient.assigned_vamcs.keys, :prompt => "Select one"     
+          = f.input :assigned_vamc, :label => "Assigned VAMC", collection: Patient.collections[:assigned_vamc], :prompt => "Select one"     
```

Comment:
this better way of doing collections makes the dropdowns only display as text fields. This is sad. Will keep looking into why.